roles/jd-server/src/lib/mod.rs

```diff
@@ -55,6 +182,15 @@ pub struct CoinbaseOutput {
     output_script_value: String,
 }
 
+impl CoinbaseOutput {
```

Comment:
saw the same struct in JDC, would be a good issue to extract elsewhere into a shared location.